policyengine_api/ai_prompts/simulation_analysis_prompt.py

```diff
@@ -0,0 +1,131 @@
+from policyengine_api.ai_templates.simulation_analysis_template import (
+    simulation_analysis_template,
+    audience_descriptions,
+)
+from pydantic import BaseModel
+from typing import Any, Annotated
+import json
+
+
+class InboundParameters(BaseModel):
+    currency: str
+    country_id: str
+    selected_version: str
+    time_period: str
+    impact: dict[str, dict[str, Any]]
+    policy_label: str
+    policy: dict[str, Any]
+    region: str
+    relevant_parameter_baseline_values: list[dict[str, Any]]
+    relevant_parameters: list[dict[str, Any]]
+    audience: str
+
+
+class AllParameters(InboundParameters):
+    model_config = {"exclude": {"audience"}}
+    enhanced_cps_template: str
+    dialect: str
+    data_source: str
+    poverty_measure: str
+    poverty_rate_change_text: str
+    poverty_by_race_text: str
+    audience_description: str
+    country_id_uppercase: Annotated[str, "Uppercase two-letter country ID"]
+    impact_budget: Annotated[str, "JSON deserialized to string"]
+    impact_intra_decile: Annotated[str, "JSON deserialized to string"]
+    impact_decile: Annotated[str, "JSON deserialized to string"]
+    impact_inequality: Annotated[str, "JSON deserialized to string"]
+    impact_poverty: Annotated[str, "JSON deserialized to string"]
+    impact_deep_poverty: Annotated[str, "JSON deserialized to string"]
+    impact_poverty_by_gender: Annotated[str, "JSON deserialized to string"]
+
+
+def generate_simulation_analysis_prompt(params: InboundParameters) -> str:
+    """
+    Generate AI prompt for economy-wide simulations
+    """
+
+    parameters: InboundParameters = InboundParameters.model_validate(params)
+
+    enhanced_cps_template: str = (
+        """Explicitly mention that this analysis uses PolicyEngine Enhanced CPS, constructed 
+    from the 2022 Current Population Survey and the 2015 IRS Public Use File, and calibrated 
+    to tax, benefit, income, and demographic aggregates."""
+        if "enahnaced_us" in parameters.region
```

Comment:
I'm guessing...can we test this?
```suggestion
        if "enhanced_us" in parameters.region
```

Reply:
You're right, I've never seen "enhanced" spelled so weirdly. I've added a test, filed and fixed a related issue (https://github.com/PolicyEngine/policyengine-api/issues/2172), added a test for that issue, and made some corrections to some fixtures.